### DISCUSSION TITLE
Error while adding scraped users in the grouo

### DISCUSSION THREAD

**Discussion by kutanoor** [JohnMiltonHacks/telegram-scraper#3] [Q&A] [answered]
Getting Flood Error from telegram. Script is stopping now. Please try again after some time.
Traceback (most recent call last):
  File "/data/data/com.termux/files/home/telegram-scraper/sim.py", line 92, in <module>
    api_id = data[t]
IndexError: list index out of range

I am getting this error while adding scraped users to my group.can you please resolve the issue and release an update.

**Comment by JohnMiltonHacks** [comment, marked as answer]
> Getting Flood Error from telegram. Script is stopping now. Please try again after some time. Traceback (most recent call last): File "/data/data/com.termux/files/home/telegram-scraper/sim.py", line 92, in api_id = data[t] IndexError: list index out of range
> 
> I am getting this error while adding scraped users to my group.can you please resolve the issue and release an update.

Updated
